### DISCUSSION TITLE
SFS now in sklearn

### DISCUSSION THREAD

**Discussion by PadraigC** [rasbt/mlxtend#798] [Q&A] [answered]
I see Sequential Feature Selection has now turned up in the latest sklearn distribution. 
Is this something that is moving from mlxtend into sklearn?
It has less features than the mlxtend version, in particular the **get_metric_dict()** method.

**Comment by rasbt** [comment, marked as answer]
Yes, this is indeed based on the version implemented in scikit-learn. We began porting it back in 2017 (https://github.com/scikit-learn/scikit-learn/pull/8684) and it took some time implementing. Indeed it was planned as a simplified version (originally just forward selection). In any case, I will keep the current SFS version also in mlxtend. There may actually be some improvements I have in mind in case I ever find some extra time for this. 

I actually just collected those in the Projects tab to better keep track of this: https://github.com/rasbt/mlxtend/projects/6

**Comment by PadraigC** [comment]
Thank you for taking the time to clarify this.

Pádraig.
